lib/utils/format.js

```diff
@@ -434,26 +435,35 @@
             className += WHITESPACE + /*'unipika-' + */'special-key';
         }
 
+        if (node.$incomplete && node.$original_value) {
+            title = node.$original_value;
+        }
+
         return settings.asHTML ?
-        '<span class="' + className + '">' + formattedValue + '</span>' :
+        '<span' + (title ? ' title="' + title + '"' : '') + ' class="' + className + '">' + formattedValue + '</span>' :
```

Comment:
As I know `settings.maxStringSize` is used when you need to do your DOM-tree lighter to make your page faster. But current approach makes DOM-tree even bigger than without the options. Will it affect performance when`$original_value` is really big?

Reply:
From performance point of view, there is not much difference, 50 characters you have in a title or 1000. `maxStringsSize` was added primarily to improve the visual representation of data, not performance